src/main/java/com/zufar/icedlatte/email/api/token/TokenCache.java

```diff
@@ -36,18 +36,4 @@ public UserRegistrationRequest getToken(String tokenKey) {
     public void removeToken(String tokenKey) {
         tokenCache.invalidate(tokenKey);
     }
-
-    // for test only
-    public String getToken(UserRegistrationRequest request) {
-        return getKeyByValue(tokenCache, request);
-    }
-
```

Comment:
I'm afraid this method was requested by QA , we should probably discuss it with Tanya